setup.cfg

```diff
@@ -100,10 +100,16 @@ doctest_plus = enabled
 text_file_format = rst
 open_files_ignore = "astropy.log" "/etc/hosts" "*.ttf"
 remote_data_strict = true
-addopts = -p no:warnings --doctest-rst
+addopts = --doctest-rst
 asdf_schema_root = astropy/io/misc/asdf/data/schemas
 xfail_strict = true
 qt_no_exception_capture = 1
+filterwarnings =
+    ignore:unclosed file:ResourceWarning
+    ignore:unclosed <socket:ResourceWarning
+    ignore:unclosed <ssl.SSLSocket:ResourceWarning
+    ignore:The timefunc function is deprecated
+    ignore:numpy.ufunc size changed:RuntimeWarning
```

Comment:
@mhvk , should we worry about `numpy.ufunc size changed` or can we ignore it? I wonder if it is a complain that is exclusive to `conda`.

Reply:
I think we should indeed ignore this, it can happen with pip too in some cases, but it's beyond the scope of our test suite.

Reply:
Do you know why `numpy.ufunc` is tormenting us so? I am curious.

Reply:
Not fully, but it's a common issue with some combinations of numpy and other packages, it's common enough we should just always ignore it.